src/db/mod.rs

```diff
@@ -0,0 +1,23 @@
+use actix_web::actix::*;
+use actix_web::*;
+use diesel;
+use diesel::prelude::*;
+use diesel::r2d2::{ConnectionManager, Pool};
+use dotenv::dotenv;
+use std::env;
+
+pub mod handlers;
+pub mod messages;
+pub mod models;
+
+pub type ConnectionType = SqliteConnection;
+
+pub struct DbExecutor(pub Pool<ConnectionManager<ConnectionType>>);
+
+impl Actor for DbExecutor {
+    type Context = SyncContext<Self>;
+}
+
+pub fn create_manager() -> ConnectionManager<ConnectionType> {
+    ConnectionManager::<SqliteConnection>::new("app2.db")
```

Comment:
Should this be using the environment variable?

Reply:
Yes it probably should be. We can do this a couple ways. We can use dotenv files. There is a handy rust crate for this. We could also put it in the configuration. Do yo have an opinion?

Reply:
For history...we decided to worry about this problem in a different PR.